siac/main.go

```diff
@@ -179,7 +179,7 @@ func main() {
 
 	root.AddCommand(walletCmd)
 	walletCmd.AddCommand(walletAddressCmd, walletSendCmd, walletSiafundsCmd, walletStatusCmd, walletUnlockCmd, walletLockCmd, walletInitCmd, walletAddseedCmd, walletSeedsCmd)
-	walletInitCmd.Flags().StringVarP(&initPassword, "password", "p", "", "Encryption password used for wallet")
+	walletInitCmd.Flags().BoolVarP(&initPassword, "password", "p", false, "Encryption password used for wallet")
```

Comment:
you only need to supply a password in 3 commands (encrypt, unlock, recover). Encrypt I think by default should just use a blank password, and the others are required to use a password.

So the flag is probably not needed at all.